.vib/jenkins/cypress/cypress/integration/jenkins_spec.js

```diff
@@ -0,0 +1,94 @@
+/// <reference types="cypress" />
+import { random, checkErrors } from './utils';
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('.alert-danger').should('not.exist');
+  cy.get('div#tasks');
+
+  cy.get('[href$="/logout"]').click();
+  cy.get('form[name*="login"]');
+});
+
+it('should be possible to create a new Git Project', () => {
```

Comment:
It would be better to call this test like this:
`it('should be possible to create a new Jenkins job')`
or 
`it('should be possible to create a new Jenkins pipeline')`

Reasoning: We are not exactly creating a Git Project. We are creating a Jenkins job from a Github project.